pkg/kube/pod_runner.go

```diff
@@ -60,14 +55,7 @@ func NewPodRunnerWithPodController(pc PodController) PodRunner {
 }
 
 // Run will create a new Pod based on PodRunner contents and execute the given function
-func (p *podRunner) Run(ctx context.Context, fn func(context.Context, *v1.Pod) (map[string]interface{}, error)) (map[string]interface{}, error) {
-	return p.RunEx(ctx, func(innerCtx context.Context, pc PodController) (map[string]interface{}, error) {
-		return fn(innerCtx, pc.Pod())
-	})
-}
-
-// RunEx will create a new Pod based on PodRunner contents and execute the given function
-func (p *podRunner) RunEx(ctx context.Context, fn func(context.Context, PodController) (map[string]interface{}, error)) (map[string]interface{}, error) {
+func (p *podRunner) Run(ctx context.Context, fn func(context.Context, PodController) (map[string]interface{}, error)) (map[string]interface{}, error) {
```

Comment:
```suggestion
func (p *podRunner) Run(
	ctx context.Context,
	fn func(context.Context, PodController) (map[string]interface{}, error),
) (map[string]interface{}, error) {
```
[nit] improve readability